doc/sql.txt

```diff
@@ -0,0 +1,436 @@
+================================================================================
+                                                                       *sql.lua*
+
+SQLite/LuaJIT binding and highly opinionated wrapper for storing, retrieving,
+caching, persisting, querying, and connecting to SQLite databases.
+To find out more:
+https://github.com/tami5/sql.nvim
+  :h sql
+  :h sql.table
+
+sql.new({uri})                                                     *sql.new()*
```

Comment:
idk it feels wrong to call it sql, db might be better but that something for another pr

Reply:
You know that we then have to rename this: https://github.com/tami5/sql.nvim/blob/3a51b7b9780d76f8f2feef7e2034fb6fe4cf3e60/lua/sql.lua#L20

Reply:
yep yep makes sense to do so will do later